satinfo/msu_n14/readme

```diff
@@ -1 +1,2 @@
-n14  19950119 000000  20061001 240000  msu    4  1 2 3 4	 # end date as in CFSR (Merra off 20061010)			
+# NOAA assimilated ch1, NASA did not
```

Comment:
Jeff. where did the msu info come from? Should it be part of this PR?

Reply:
I added it since I had a little extra time